src/Library/demos/Box/main.js

```diff
@@ -0,0 +1,76 @@
+import Gtk from "gi://Gtk";
+
+const interactive_box = workbench.builder.get_object("interactive_box");
+const button_add = workbench.builder.get_object("button_add");
+const prepend = workbench.builder.get_object("prepend");
+const button_remove = workbench.builder.get_object("button_remove");
+var count = 0;
+
+button_add.connect("clicked", () => {
+  const label = new Gtk.Label({
+    name: "card",
+    label: `Item ${count + 1}`,
+  });
+  const style = label.get_style_context();
+  style.add_class("card");
```

Comment:
This should just be added as a construct property, especially since style contexts will deprecated sometime soon.

```js
const label = new Gtk.Label({
  name: 'card',
  label: `Item ${count + 1}`,
  css_classes: ['card'],
});
```
`Gtk.Label` doesn't have any CSS classes by default, but for widgets that do you can use `GtkWidget.add_css_class()` to avoid overwriting existing classes.

Reply:
I was looking for something like this, thanks!

Reply:
I had completely missed that in GTK 4

https://docs.gtk.org/gtk4/property.Widget.css-classes.html

TIL - thanks @andyholmes